src/sysinfo.cc

```diff
@@ -366,6 +366,33 @@ std::vector<CPUInfo::CacheInfo> GetCacheSizes() {
 #endif
 }
 
+std::string GetSystemName() {
+#if defined(BENCHMARK_OS_WINDOWS)
+  std::string str;
+  const unsigned COUNT = MAX_COMPUTERNAME_LENGTH+1;
+  TCHAR  hostname[COUNT];
+  DWORD DWCOUNT = COUNT;
+  if (!GetComputerName(hostname, &DWCOUNT))
+    return std::string("Unable to Get Host Name");
+#ifndef UNICODE
+  str = hostname;
+#else
+  std::wstring wStr = hostname;
+  str = std::string(wStr.begin(), wStr.end());
+#endif
+  return str;
+#else
+#ifdef BENCHMARK_OS_MACOSX //Mac Doesnt have a copy for Host Name in it
+#define HOST_NAME_MAX 64
+#endif
+  const unsigned COUNT = HOST_NAME_MAX;
```

Comment:
Can't you just use `HOST_NAME_MAX` directly, why do you need `COUNT` variable?

Reply:
Removing unnecessary COUNT variable references, using macro directly